ui/app/components/job-page/parts/job-status-panel.js

```diff
@@ -0,0 +1,10 @@
+// @ts-check
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class JobPagePartsJobStatusPanelComponent extends Component {
```

Comment:
`thought`: there doesn't seem to be an invocation of this component. Maybe we're getting mixed up and `job-status/panel` should be here?

Reply:
Yep good catch. I had made 2 and meant to get rid of the obsoleted one.